ksqldb-rocksdb-config-setter/src/main/java/io/confluent/ksql/rocksdb/KsqlBoundedMemoryRocksDBConfig.java

```diff
@@ -92,7 +119,33 @@ public class KsqlBoundedMemoryRocksDBConfig extends AbstractConfig {
           Type.DOUBLE,
           INDEX_FILTER_BLOCK_RATIO_DEFAULT,
           Importance.LOW,
-          INDEX_FILTER_BLOCK_RATIO_DOC
+          INDEX_FILTER_BLOCK_RATIO_DOC)
+      .define(
+          COMPACTION_STYLE,
+          Type.STRING,
+          CompactionStyle.UNIVERSAL.name(),
+          ConfigDef.ValidString.in(
+              CompactionStyle.UNIVERSAL.name(), CompactionStyle.LEVEL.name()),
+          Importance.LOW,
+          COMPACTION_STYLE_DOC)
+      .define(
+          COMPRESSION_TYPE,
+          Type.STRING,
+          CompressionType.NO_COMPRESSION.name(),
+          ConfigValidators.enumValues(CompressionType.class),
+          Importance.LOW,
+          COMPRESSION_TYPE_DOC
+      ).define(
+          MAX_BACKGROUND_JOBS,
+          Type.INT,
+          -1,
+          Importance.LOW,
+          MAX_BACKGROUND_JOBS_DOC
+      ).define(
+          COMPACTION_TRIVIAL_MOVE,
+          Type.BOOLEAN,
```

Comment:
Why was the default value for this config removed? I think that means it's required now, and that's why your unit tests such as `shouldDefaultTrivialMove()` are failing.